aidial_interceptors_sdk/examples/app_factory.py

```diff
@@ -0,0 +1,39 @@
+from aidial_sdk import DIALApp
+from aidial_sdk.telemetry.types import TelemetryConfig
+
+from aidial_interceptors_sdk.chat_completion import (
+    interceptor_to_chat_completion,
+)
+from aidial_interceptors_sdk.embeddings.adapter import interceptor_to_embeddings
+from aidial_interceptors_sdk.examples.registry import Interceptors
+from aidial_interceptors_sdk.examples.utils.log_config import configure_loggers
+from aidial_interceptors_sdk.utils._http_client import HTTPClientFactory
+
+
+def create_app(
```

Comment:
Because the examples server isn't part of tests, it's part of the package and is enabled by the extras "examples":

https://github.com/epam/ai-dial-interceptors-sdk/?tab=readme-ov-file#from-package